hsapp/menubar.go

```diff
@@ -58,6 +58,7 @@ func (a *App) renderMainMenuBar() {
 					}
 				}),
 			}),
+			g.MenuItem("Close Project").OnClick(a.onExitProjectClicked),
```

Comment:
lets add `Enabled( a.project != nil)`

Reply:
Thank you for pointing that out! I made the necessary changes.